igor-web/src/main/groovy/com/netflix/spinnaker/igor/scm/stash/client/model/TextLinesResponse.groovy

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.igor.scm.stash.client.model
+
+import java.util.stream.Collectors
+
+class TextLinesResponse extends AbstractStashResponse {
+  List<Map<String, String>> lines
```

Comment:
hmm maybe it's better to set a default value for `lines` and then  remove     if (lines == null) {
      return "null" ? I'm not a fan of this check :)